.circleci/config.yml

```diff
@@ -173,7 +173,7 @@ jobs:
             - out
       - run:
           name: Lint examples
-          command: yarn run lint:examples
+          command: npm i --only=dev && npm run lint:examples
```

Comment:
We are actually very intentionally not installing all of the dev dependencies here and only installing the bare minimum to speed up the time it takes the ci to run. That's why the global add on line 160 is there.

Reply:
Hmm I thought that might be the case, just wanted to get something working first. I think we will need the eslint modules installed to run the lint though.